integration/helpers/createTree.js

```diff
@@ -0,0 +1,58 @@
+const { v4: uuid } = require('uuid')
+
+/**
+ * This function is used to create the aggregation tree
+ *
+ * @param {Webhooks} querierWebhooks The webhooks of the querier
+ * @param {Webhooks[]} aggregatorsWebhooks The list of webhooks used by aggregators
+ * @param {Webhooks[]} contributorsWebhooks The list of webhooks used by contributors
+ * @returns 
+ */
+const createTree = (querierWebhooks, aggregatorsWebhooks, contributorsWebhooks) => {
+  // TODO: Make a dynamic tree
+  const querier = {
+    webhook: querierWebhooks.aggregationWebhook,
+    level: 0,
+    nbChild: 3,
+    aggregatorId: uuid(),
+    finalize: true
+  }
+
+  const aggregators = [
+    {
+      webhook: aggregatorsWebhooks[0].aggregationWebhook,
+      level: 1,
+      nbChild: contributorsWebhooks.length,
+      parent: querier,
+      aggregatorId: uuid(),
+      finalize: false
+    },
+    {
+      webhook: aggregatorsWebhooks[1].aggregationWebhook,
+      level: 1,
+      nbChild: contributorsWebhooks.length,
+      parent: querier,
+      aggregatorId: uuid(),
+      finalize: false
+    },
+    {
+      webhook: aggregatorsWebhooks[2].aggregationWebhook,
+      level: 1,
+      nbChild: contributorsWebhooks.length,
+      parent: querier,
+      aggregatorId: uuid(),
+      finalize: false
+    }
+  ]
+
+  let contributors = contributorsWebhooks.map(e => ({
+    ...e,
+    level: 2,
+    nbChild: 0,
+    parents: aggregators
+  }))
+
+  return contributors
+}
+
+module.exports = createTree
```

Comment:
Isn't the exact same file than https://github.com/JulienMirval/dissec_cozy/blob/master/scripts/helpers/createTree.js ? :thinking: 
We should never duplicate files like this, you should declare it once and export the methods you need to use it where necessary

Reply:
I'm unsure where the function belongs: is it a script helper or an integration test helper?

Reply:
Maybe in a `src/lib/tree.js` file  ?